### PR TITLE
Fix 500 error when attempting to update installed repository.

### PR DESCRIPTION
**Pull request by davebx**

### CODE REVIEW

File: lib/galaxy/util/__init__.py [hunk 1]
```diff
@@ -335,6 +335,17 @@ def shrink_string_by_size( value, size, join_by="..", left_larger=True, beginnin
     return value
 
 
+def parse_query_string( query_string ):
```
Comment (nsoranzo) [inline lib/galaxy/util/__init__.py]: Should this function be able to deal with malicious `query_string`s ? E.g.: `'foo=3&bar=4&'`, `'foo&pluto=4'`

Comment (davebx) [inline lib/galaxy/util/__init__.py]: I don't think it will be necessary, but there's absolutely no harm in preventing malice.

Comment (nsoranzo) [inline lib/galaxy/util/__init__.py]: Brilliant, thanks! I think that now that you've added `if '=' not in argument`, you can remove:

```
    if '=' not in query_string:
        return {}
```

Comment (davebx) [inline lib/galaxy/util/__init__.py]: @carlfeberhard reminded me that urlparse.parse_qs exists.

**Comment by martenson** [comment]
is there an example of repo that has this problem? @bgruening @davebx

File: lib/galaxy/util/__init__.py (hunk 1 above)
Comment (nsoranzo) [inline lib/galaxy/util/__init__.py]: Good idea!

**Comment by davebx** [comment]
@martenson I've been able to reliably cause the error locally, but not when installing or updating from TTS.

**Comment by yhoogstrate** [comment]
This PR solves the issue for me, thanks @davebx !

**Comment by bgruening** [comment]
@davebx @martenson this is a bugfix can we get it into 15.10?

**Comment by martenson** [comment]
@bgruening I am working on that already, stay tuned! :)

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**